benchmarks/common.py

```diff
@@ -1449,14 +1452,35 @@ def main(runner, original_dir=None):
     experiment = functools.partial(experiment, args, model_iter_fn)
 
     if args.only:
```

Comment:
how does this work when args.only is false?

Reply:
Well, if you're running with `isolate` (which I think we always should), it just ends up calling `--only`.

Reply:
Can we check then that if `--batch-size-file` is set, either `--isolate` or `--only` is set too?

Reply:
Updated.